docs/source/general/abstract_models.rst

```diff
@@ -2,20 +2,22 @@
 Abstract Models
 ===============
 
-Firstly, we need to add basic models. TimeStampedEditableModel is an abstract base class model that provides self-updating
-created and modified fields. If we write the base class and put abstract=True in the Meta class, this model will then not be used to
-create any database table. Instead, when it is used as a base class for other models, its fields will be added to those
-of the child class.
+Firstly, we need to add basic models. TimeStampedEditableModel is an abstract base class model, imported from openwisp_utils that provides self-updating
+created and modified fields. 
+
+.. note::
+   An abstract class model is not used to create any database table, but is used as a base class for other models. 
+   Its fields will be added to those of the child class. Assigning ``abstract=True`` in Meta class, makes that class abstract. 
```

Comment:
TimeStampedEditableModel` has been removed from django-freeradius because it's included in `openwisp-utils` and it is not needed to explain it here. We can remove everything and focus on explaining the base/abstract models of django-freeradius and how to create a customized app based on django-freeradius.